pyfarm/agent/http/api/state.py

```diff
@@ -46,6 +53,15 @@ def post(self, **kwargs):
         request = kwargs["request"]
         data = kwargs["data"]
         agent = config["agent"]
+
+        try:
+            os.remove(config["run_control_file"])
+        except (WindowsError, OSError, IOError) as e:
+            if e.errno != ENOENT:
+                logger.error("Could not delete run control file %s: %s: %s",
+                             config["run_control_file"],
+                             type(e).__name__, e)
```

Comment:
Probably should add a call to `atexit.register` here to try and clean the file when Python exits.